blog/_posts/2023-07-21-2023-nextstrain-hiring.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Openings for bioinformatician and software engineer to contribute to Nextstrain platform
+author: Trevor Bedford
+---
+
+Positions for a __bioinformatician__ and a __software engineer__ are available immediately in the Bedford lab at the Fred Hutch. Details for both positions follow:
+
+-----------------------------------------
+
+### Bioinformatics Analyst II/III
+
+We have an opening for a bioinformatician in the [Bedford lab](https://bedford.io/) at the Fred Hutch to work on genomic epidemiology and evolutionary analysis of pathogens such as SARS-CoV-2, seasonal influenza, and [other emerging and endemic pathogens](https://nextstrain.org/pathogens). This position will contribute to ongoing work for the Bedford lab and Nextstrain.
+
+[Nextstrain](https://nextstrain.org/) is an award-winning project for tracking infectious disease epidemics developed in collaboration with the [Neher lab](https://neherlab.org/) at the University of Basel. Nextstrain won the [Open Science Prize](https://www.nih.gov/news-events/news-releases/open-science-prize-announces-epidemic-tracking-tool-grand-prize-winner) in Feb 2017 and has been instrumental in analysis of the [SARS-CoV-2 pandemic](https://nextstrain.org/sars-cov-2), [Ebola outbreaks](https://nextstrain.org/community/inrb-drc/ebola-nord-kivu), [Zika spread](https://nextstrain.org/zika/) in the Americas and is used by the World Health Organization to aid in the process of [influenza vaccine strain selection](https://nextstrain.org/flu/). The software we write to power [all parts](https://docs.nextstrain.org/en/latest/learn/parts.html) of Nextstrain---bioinformatics, visualizations, analysis pipelines, data management, and more---is entirely [open-source](https://opensource.org/osd) and [available to the public](https://github.com/nextstrain). We work with public health entities and scientists across the world, both formally and informally, to expand pathogen surveillance capabilities and to improve the automation and robustness of these analyses. Our goal is to empower the wider genomic epidemiology and public health communities to tweak our analyses, create new ones, and communicate scientific insights using the same tools we do.
```

Comment:
I wish we had standardized these intro blurbs for both job descriptions. If we had, we could place a single version at the top of the blog post to avoid the redundancy here and below. As it stands, maybe it's better to keep the separate versions since people will scroll to the job title they are interested in anyway?

Reply:
Yeah, it's obvious that they're conveying the same thing with different wording. Separate versions should be fine, I [added section links](https://github.com/blab/blotter/pull/34#discussion_r1271154134) which should help.